test/integration/admin/api_docs/account_api_docs_controller_test.rb

```diff
@@ -216,6 +172,11 @@ class MasterLoggedInTest < Admin::ApiDocs::AccountApiDocsControllerTest
 
   private
 
+  def assert_account_active_docs_menus
+    expected_active_menus = {main_menu: :audience, submenu: :cms, sidebar: :ActiveDocs, topmenu: :dashboard}
```

Comment:
we are using topmenu?

Reply:
Right :+1: Probably not. I will check that and change it if not :+1: